cmd/abapEnvironmentCheckoutBranch.go

```diff
@@ -122,9 +109,9 @@ func triggerCheckout(repositoryName string, branchName string, checkoutConnectio
 	defer resp.Body.Close()
 
 	// workaround until golang version 1.16 is used
-	time.Sleep(1 * time.Second)
+	time.Sleep(100 * time.Millisecond)
```

Comment:
100 Milliseconds = 0.1 second, is this change intended?

Reply:
yes, I think less time is sufficient.